proxy/tests/telemetry.rs

```diff
@@ -75,18 +75,18 @@ impl Fixture {
 }
 
 impl TcpFixture {
-    fn server() -> server::Listening {
-        let msg1 = "custom tcp hello";
-        let msg2 = "custom tcp bye";
+    const HELLO_MSG: &'static str = "custom tcp hello";
+    const BYE_MSG: &'static str = "custom tcp bye";
```

Comment:
thanks, i was planning on doing this and forgot 👍 

sidenote: these same messages are used in `tests::transparency` as well, maybe the consts should live in the test support module and the transparency tests should also use them??